pkg/config/msp.go

```diff
@@ -670,6 +670,24 @@ func (m *MSP) validateCACerts() error {
 	if err != nil {
 		return fmt.Errorf("invalid intermediate cert: %v", err)
 	}
+	rootCertPool := x509.NewCertPool()
```

Comment:
I know we'll use this in the future commit related to the TODO below, but we shouldn't get in a habit of leaving around code we aren't using. :)

Reply:
goot catch, thanks.